packages/webpack-cli/lib/utils/cli-executer.js

```diff
@@ -50,10 +50,14 @@ async function prompter() {
 }
 
 async function run() {
-    const args = await prompter();
-    process.stdout.write('\n');
-    logger.info(`Executing CLI\n`);
-    runner([], args);
+    try {
+      const args = await prompter();
+      process.stdout.write('\n');
+      logger.info(`Executing CLI\n`);
+      runner([], args);	
+    } catch (err) {
+      logger.error('Interrupted')
```

Comment:
```suggestion
      logger.error('Action Interrupted, use `webpack-cli help` to see possible options.')
```

Maybe this? What say @ematipico

Reply:
Much better! :)

Reply:
We can wrap `webpack-cli help` in chalk to colorize it maybe, that will be even better.